src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +47,10 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        return numbers.stream()
+                .filter((val) -> val > 3)
+                .mapToLong((val) -> val * 2)
+                .reduce((val1, val2) -> val1 + val2)
+                .orElse(0L);
```

Comment:
```suggestion
                .reduce(0L, Long::sum);
```
이렇게 하면 orElse를 해줄 필요가 없답니다!

Reply:
꿀팁 감사합니다